packages/block-library/src/image/edit.native.js

```diff
@@ -153,13 +151,6 @@ export class ImageEdit extends Component {
 		) {
 			mediaUploadSync();
 		}
-
-		// Flag the ID of a post's featured image when editor mounts.
-		if ( attributes.id ) {
-			setAttributes( { featuredImageId } );
```

Comment:
Good to see this removal Siobhan! I've been meaning to comment on the PR for it, in the sense that the Image component shouldn't keep `featuredImageId` in its attribute list as it's not an attribute of the block itself but rather, a property of the global post object. 👍

Reply:
Thank you!